### ISSUE TITLE
[Calendar] Year and month buttons with somewhat unclear names

### ISSUE THREAD

**Issue by Magsu** [sl-design-system/components#2938]
### Provide a general summary of the issue here

This is not a violation, but rather a suggestion for improvement.

The calendar has buttons for a year and month selector. Currently their names are the current year number (e.g., "2025") and the current month (e.g., "January"). The names could be clearer of indicating what action the button performs.

<img width="266" height="67" alt="Image" src="https://github.com/user-attachments/assets/ba22c03e-fc71-4ba7-a0ac-415dbfe2cceb" />

### 🤔 Expected Behavior?

Users have a clear understanding of what the buttons actually do.

### 😯 Current Behavior

Users can have an understanding of what actions the buttons represent, but it might not be clear to users who don't frequently use calendars.

### 💁 Possible Solution

Add the action to the label. E.g., "2025. Select year" and "January. Select month".

### 🖥️ Steps to Reproduce

-

### What browsers are you seeing the problem on? (only for bugs in code)

Chrome

### What operating system are you using?

Windows 11

### 👤 Your name

Valtteri Järvinen

### 🧢 Your product/team

External

### 🎨 Your Theme(s)

Sanoma Learning
